docs/job.md

```diff
@@ -25,19 +25,25 @@ In the application, each job is represented as an object.  Regardless of type, a
 
 ### `DownloadJob`
 
-Download jobs represent work to download attachments or file formats, both of which are specified in the results of a query to the `document` endpoint.  For example:
+Download jobs represent work to download attachments or file formats, both of which are specified in the results of a query to the `document` endpoint. As the information of the files location can not be retrieved from the endpoint itself, it is stored in the job from when it comes from the document endpoint.
+* `url` - the URL of the attachment or file format
+* `folder_name` - the full name of the path to where the file will be placed
+* `file_name` - the name of the file in which is will be placed
+  * Can be a variety of file types: html, pdf, etc.
```

Comment:
Should we specify here that file_type is actually a field in the DownloadJob? It seems to me like it should be similar to how the other three fields are typed above it.